kafka/channel/config/200-dispatcher-clusterrole.yaml

```diff
@@ -68,3 +68,14 @@ rules:
     verbs:
       - create
       - patch
+  - apiGroups:
+      - "coordination.k8s.io"
+    resources:
+      - leases
+    verbs:
```

Comment:
Why is this a smaller list of verbs than elsewhere (missing `delete`)?

Reply:
Inadvertent, I was tired.